flaskutils/models.py

```diff
@@ -1,8 +0,0 @@
-from pgsqlutils.orm import BaseModel
-
-
-class FlaskModel(BaseModel):
```

Comment:
Grand, I see. So, if we're working on projects that use `flaskutils` as a dependency, should we wait for this refactoring to take place before rebasing the latest `flaskutils` changes?